doc/api.yml

```diff
@@ -0,0 +1,783 @@
+          $ref: '#/components/responses/500'
+
+  '/posts/{post_id}/comments/{id}':
+    get:
+      tags:
+        - Comments
+      summary: Get comment by identifier.
+      operationId: getCommentById
+      parameters:
+        - name: post_id
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Comment by identifier.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Comment'
+        '400':
+          $ref: '#/components/responses/400'
+        '403':
+          $ref: '#/components/responses/403'
+        '404':
+          $ref: '#/components/responses/404'
+        '429':
+          $ref: '#/components/responses/429'
+        '500':
+          $ref: '#/components/responses/500'
+
+components:
+  securitySchemes:
+    BearerAuth:
+      type: http
+      scheme: bearer
+
+  responses:
+    ## response model
+    '400':
+      description: Bad request
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/BadRequestException'
+    '403':
+      description: Forbidden
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/NotAuthorizedException'
+    '404':
+      description: Not found
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/ResourceNotFoundException'
+    '429':
+      description: Too many requests to the API
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/TooManyRequestsException'
+    '500':
+      description: Internal server error
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/InternalServerException'
+
+  schemas:
+    ## enum type
+    CategoryType:
+      type: string
+      enum:
+        - STUDY
+        - IT
+        - SCIENCE
+        - MATHEMATICS
+        - BOOKS
+        - ELSE
+    FollowType:
+      type: string
+      enum:
+        - FOLLOW
+        - UNFOLLOW
+    Page:
+      type: integer
+      description: Start at `1`
+    PageSize:
+      type: integer
+      description: Cannot exceed `500`
+    Sex:
+      type: string
+      enum:
+        - M
+        - F
+    ReactionType:
+      type: string
+      enum:
+        - LIKE
+        - DISLIKE
+
+        ## resource model
+    Post:
+      properties:
+        id:
+          type: string
+        picture_url:
+          type: string
+        description:
+          type: string
+        content:
+          type: string
+        title:
+          type: string
+        creation_datetime:
+          type: string
+          format: date-time
+    Comment:
+      properties:
+        id:
+          type: string
+        post:
```

Comment:
We'll only need the associated post_id, we won't need the entire `post`
We see the post before the comment so